include/openssl/ssl.h

```diff
@@ -2184,7 +2191,7 @@ void SSL_CTX_set0_ctlog_store(SSL_CTX *ctx, CTLOG_STORE *logs);
  */
 const CTLOG_STORE *SSL_CTX_get0_ctlog_store(const SSL_CTX *ctx);
 
-# endif /* OPENSSL_NO_CT */
+# endif                         /* OPENSSL_NO_CT */
```

Comment:
This seems to be the only comment on ```#endif``` in the file; I almost wonder if it is "more consistent" with the current style to remove it than to reindent it.

Reply:
This change comes from the `openssl-format-source` tool. I don't know the reason/rule behind this change.

Reply:
@kaduk: I will be glad to remove it, if you say so.
On the other side, I'm for having this kind of comment when the length of this block exceeds the height of the screen ;)
I rebase these changes on a fresh master .

Reply:
I think the comment should remain. I don't know why the tool reindented it. I think it looks better without the reindentation.

Reply:
Fixed and rebased to last master.